### ISSUE TITLE
Running in container

### ISSUE THREAD

**Issue by scabala** [zellij-org/zellij#1057]
Hello,
first of all, I would like to sincerely thank you for all your work that made zellij real. That's really something to look up to.

I am curious if anyone is using zellij inside containerized environments? I am considering such scenario but I do not know it that's possible at all.

What I am actually doing:
 - in container, running zellij in server mode (`zellij --server /run/user/$UID/zellij/0.24.0/from-container`) with mounted socket
- on host, trying to attach to `from-container` session (`zellij attach from-container`)

I want to my development experience in containers to be "semaless" -  attaching to different (containerized) workspaces from host and run commands in containers should be doable.

I am not experienced in Rust so not able to help with coding but I am willing to test different approaches.

**Comment by jaeheonji** [comment]
Hi, @scabala! Thank you for your interest 😄 

I did some tests. and I think this is `zellij` bug. 
To find the problem, It will take time but thank you for the report!

**Comment by scabala** [comment]
Hi @jaeheonji 
thanks for the rapid reply!
I did not mention this but it would be also beneficial to include some flag for `zellij` server mode lik `--foreground` or `--no-daemon` to not fork into background. That would make  running zellij as main process in container much easier. Should I create different issue for it?

**Comment by jaeheonji** [comment]
Yes! That's a good idea. If you make it a feature Issue, I'll label it 😄

**Comment by scabala** [comment]
Cool, this is the issue: https://github.com/zellij-org/zellij/issues/1062 :smile:

**Comment by queil** [comment]
I got sort of sessions working in containers by settings the `XDG_RUNTIME_DIR=/tmp` globally in the container image. I don't manually run `zellij --server` but in my exec I just have `["bash", "-c", "zellij --session rooz || zellij attach rooz"]` - so it either starts a session or it attaches if it already exists where I enter the container.
